### PR TITLE
added environment on vertical to project cards

### PR DESCRIPTION
**Pull request by DanielekPark**
fixes issue #894
Added environment to vertical on tdm-calculator.md file on line 76, to public-tree-map.md on line 58, and to metro-ontime.md on line 58

### CODE REVIEW

File: _projects/metro-ontime.md [hunk 1]
```diff
@@ -26,11 +26,12 @@ links:
...
     url: 'https://observablehq.com/@ctsexton/railstats-marey-diagram'
-location: 
+location:
   - Downtown LA
   - Remote
 partner: LA Metro (https://www.metro.net/)
 tools: Docker, AWS, Observable
 visible: true
 status: Completed
+vertical: Environment
```
Comment (drubgrubby) [inline _projects/metro-ontime.md]: The word Environment should be contained in single quotes: 'Environment'.  This was not clear in the issue, but it is the standard.  (I'm going to edit the issue now to show that).

File: _projects/public-tree-map.md [hunk 2]
```diff
@@ -55,4 +55,5 @@ partner: City of Santa Monica
 tools: Zeplin
 visible: true
 status: Active
+vertical: Environment
```
Comment (drubgrubby) [inline _projects/public-tree-map.md]: Please change to: 'Environment'

File: _projects/tdm-calculator.md [hunk 3]
```diff
@@ -72,6 +72,6 @@ location:
   - Remote
 partner: LA Department of Transportation LADOT (https://ladot.lacity.org/)
 visible: true
-vertical: 'Under Development'
+vertical: Environment
```
Comment (drubgrubby) [inline _projects/tdm-calculator.md]: Please change to: 'Environment'

**Review by drubgrubby — CHANGES_REQUESTED** [review]
You did this exactly right.  Good job. However, the issue wasn't quite clear that it should have single quotes around 'Environment'.   If you can add the single quote marks it will be perfect.

**Comment by drubgrubby** [comment]
Also, the top line should read: "Fixes #894" , without the word "issue".

**Comment by DanielekPark** [comment]
Fixes #894 added quotes to Environment for metro-ontime.md on line 36, public-tree-map.md on line 58, and on line 76 to td-calculator.md

**Comment by drubgrubby** [comment]
@DanielekPark - Hey Daniel.  I only see the original commit on this PR; I don't see the changes where you added the quotes.   Usually if you push to the same branch it will add it to the PR.  Can you check to see if pushed to the same branch as your previous commit, or if you pushed to a new branch, or perhaps haven't pushed the changes yet?  We can look at this on Sunday if it's not clear.

**Comment by drubgrubby** [comment]
Daniel and I discussed this on Slack.  He has completed the changes, but seems to have pushed it to a new branch so it's not showing up here.  We will go over this at the dev meeting on Sunday.

**Pull request merged**